target/linux/ramips/dts/mt7620n_zyxel_lte3302.dts

```diff
@@ -0,0 +1,150 @@
+#include "mt7620n.dtsi"

```

Comment:
Please don't use Windows OS CRLF newline style. And it's better to add a license `// SPDX-License-Identifier: GPL-2.0-or-later OR MIT`.